### PR TITLE
Add  link to Appveyor status (was just an image)

### LINKED ISSUE #468: Appveyor link in README
The appveyor link in README.md is set to the image, not to appveyor

### PR DESCRIPTION
**Pull request by p-l-**
Fixes #468.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Scapy #
 
 [![Travis Build Status](https://travis-ci.org/secdev/scapy.svg?branch=master)](https://travis-ci.org/secdev/scapy)
-![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/secdev/scapy?svg=true)
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/secdev/scapy?svg=true)](https://ci.appveyor.com/project/p-l-/scapy)
```
Comment (guedou) [inline README.md]: Do you know it is `p-l/scapy` instead of `secdev/scapy `?

Comment (p-l-) [inline README.md]: `secdev/scapy` does not work.

**Comment by guedou** [comment]
Did we mess our AppVeyor config on github ?

**Comment by p-l-** [comment]
No, I've never been able to access `/secdev/scapy`, and AppVeyor won't create a project under `secdev` (or I've not been able to do so, at least).

**Comment by guedou** [comment]
Got it =)

**Pull request merged**